plugins/@grouparoo/bigquery/src/lib/connect.ts

```diff
@@ -23,6 +23,7 @@ export const connect: ConnectPluginAppMethod = async ({ appOptions }) => {
     options: Parameters<typeof bqClient.query>[0],
     callback?: Parameters<typeof bqClient.query>[1]
   ) => {
+    // @ts-expect-error the BQ types think 'options' is a string, but it's an object
     log(`[ bigquery ] ${options.query}`, "debug", options.params);
```

Comment:
This is a shame, but everything /works/ fine, but the types are a little messed up in this version

Reply:
it's just for the logger we have... so it seems low-risk